announcement/announcement-tool/tool/src/webapp/vm/announcement/chef_announcements-revise.vm

```diff
@@ -1,5 +1,8 @@
 ## $Header: /cvs/sakai2/legacy/tools/src/webapp/vm/announcement/chef_announcements-revise.vm,v 1.4 2005/05/15 04:54:19 gsilver.umich.edu Exp $ 
 <script type="text/javascript" src="/library/js/spinner.js$!{portalCdnQuery}"></script>
+<script type="text/javascript">includeLatestJQuery('announcements');</script>
+#javascript("/library/js/lang-datepicker/lang-datepicker.js")
+<script type="text/javascript" src="/library/webjars/jquery-ui/1.12.1/jquery-ui.min.js"></script>
```

Comment:
We shouldn't be embedding the jquery version numbers in script imports in the tools, if possible. Maybe we could have a includeLatestJQueryUI ?